sa/sa.go

```diff
@@ -402,6 +414,46 @@ func (ssa *SQLStorageAuthority) countCertificatesByName(domain string, earliest,
 	return len(serialMap), nil
 }
 
+// countCertificatesByExactNames returns, for a single domain, the count of
+// certificates issued in the given time range for that domain. In contrast to
+// the countCertificatesByNames function subdomains are NOT considered.
+//
+// The highest count this function can return is 10,000. If there are more
+// certificates than that matching one of the provided domain names, it will return
+// TooManyCertificatesError.
+func (ssa *SQLStorageAuthority) countCertificatesByExactName(domain string, earliest, latest time.Time) (int, error) {
```

Comment:
I like the first approach of providing a constant SQL statement as an argument.

I think the second approach will make it harder to maintain backwards compatibility with what we do now using the feature flag.